File: triatlon-java/networking/src/main/java/triatlon/networking/dto/ResultDTO.java
package triatlon.networking.dto;

import triatlon.model.person.Referee;

import java.io.Serializable;

public class ResultDTO implements Serializable {

    private Referee referee;
    private Integer athleteId;
    private Integer points;

    public ResultDTO(Referee referee, Integer athleteId, Integer points) {
        this.referee = referee;
        this.athleteId = athleteId;
        this.points = points;
    }

    public ResultDTO(Integer athleteId, Integer points) {
        this.athleteId = athleteId;
        this.points = points;
    }

    public Referee getReferee() {
        return referee;
    }

    public void setReferee(Referee referee) {
        this.referee = referee;
    }

    public Integer getAthleteId() {
        return athleteId;
    }

    public void setAthleteId(Integer athleteId) {
        this.athleteId = athleteId;
    }

    public Integer getPoints() {
        return points;
    }

    public void setPoints(Integer points) {
        this.points = points;
    }

}

File: triatlon-cs/Networking/DTO/ResultDto.cs
using System;
using Model.Person;

namespace Networking.DTO
{
    [Serializable]
    public class ResultDto
    {
        private Referee _referee;
        private int _athleteId;
        private int _points;

        public ResultDto(Referee referee, int athleteId, int points)
        {
            _referee = referee;
            _athleteId = athleteId;
            _points = points;
        }

        public ResultDto(int athleteId, int points)
        {
            _athleteId = athleteId;
            _points = points;
        }
        
        public Referee Referee
        {
            get => _referee;
            set => _referee = value;
        }
        
        public int AthleteId
        {
            get => _athleteId;
            set => _athleteId = value;
        }
        
        public int Points
        {
            get => _points;
            set => _points = value;
        }
    }
}